sources/views/top.js

```diff
@@ -0,0 +1,28 @@
+import { JetView } from "webix-jet";
+import { data } from "models/menu";
+
+export default class TopView extends JetView {
+	config() {
+		const rightMenu = {
+			view: "menu",
+			localId: "top:menu",
+			layout: "y",
+			width: 200,
+			select: true,
+		};
+		const topView = {
+			cols: [
+				rightMenu,
+				{
+					$subview: true
+				}
+			]
+		};
+		return topView;
+	}
+	init() {
+		this.topview = this.$$("top:menu");
+		this.topview.parse(data);
+		this.topview.select(data.config.data[0].id);
```

Comment:
view does not know anything about data  - it just parses it (data collection is optional there, a simple array of options is also ok).

to highlight the first menu item, you can use `menu.getFirstId()`, but still if you refresh the page with `/top/data` url, you need to reflect the correct url chunk, not first:

~~~js
urlChange(view, url){
     if(url[1] && url[1].page)
         menu.select(url[1].page);
}
~~~

Reply:
done